tests/e2e/test_chunked_prefill.py

```diff
@@ -124,3 +125,61 @@ def wrapper(self, *args, **kwargs):
                     vllm_results=vllm_results,
                     hf_results=hf_outputs,
                     prompts=[prompt])
+
+
+# chunked prefill too? Or just prefix cachhing?
```

Comment:
good question, if `mode` is set to `cp` only then we shouldn't have any cache hit right (`hit_tokens / total_tokens == 0.0`)? we could test that also to ensure it is the case